incubator/bitcoind/values.yaml

```diff
@@ -0,0 +1,39 @@
+image:
+  repository: "kuberstack/bitcoind"
+  tag: "latest"
+  pullPolicy: IfNotPresent
+
+rpcPort: 443
```

Comment:
Why aren't we using the default ports? `8332,8333` and `18332,18333`. I think that needs to be added to this chart before merge.

Reply:
These settings could be added to a service section:

```
service:
    rpcport: 8332
    p2pport: 8333
    selector: tx-broadcast-svc
```